test/fixtures/property-prefix.js

```diff
@@ -13,6 +13,11 @@ const skipProperties = [
   "font-language-override",
 ]
 
+const isNotSupported = (o) =>
+    o.level === "none"
+    // http://caniuse.com/#search=object-position
+    || o.property === "object-position" && o.notes.indexOf(1) > -1
```

Comment:
Checkout the footnotes at: http://caniuse.com/#search=object-position
The number 1 corresponds to the notes in the caniuse database.